FWCore/ParameterSet/python/Config.py

```diff
@@ -707,6 +785,40 @@ def _sequencesInDependencyOrder(self):
             raise RuntimeError("circular sequence dependency discovered \n"+
                                ",".join([label for label,junk in dependencies.iteritems()]))
         return returnValue
+    def _tasksInDependencyOrder(self):
```

Comment:
Given the similarities between Tasks and Sequences in this case, is it feasible for the two to use the same code to do this check?